lib/main.dart

```diff
@@ -37,7 +37,7 @@ class MyApp extends StatelessWidget {
 class Task {
   final String title;
   bool completed;
-
+  //Majd
```

Comment:
Consider adding separate fields for task description and due date for clearer data management and better user experience